src/pages/AdminManageInstitutions.js

```diff
@@ -98,7 +98,34 @@ export default function AdminManageInstitutions(props) {
             div({style: Styles.SMALL}, ['Select and manage Institutions'])
           ]),
         ]),
-        h(SearchBar, {handleSearchChange, currentPage})
+        h(SearchBar, {handleSearchChange, currentPage,
+          style: {
+            width: '60%',
+            margin: '0 3% 0 0',
+          },
+          button: div({
+            style: {
+              display: 'flex',
```

Comment:
We can remove these styles since the item is no longer intended to be pulled to the right of the page, the search bar i think handles it

Reply:
I removed the styles under button and noticed no change but when I tried removing that div on line 106 I noticed more whitespace below the header section (see screenshot). Was that the correct div to remove or is there something else I need to adjust to prevent that whitespace there?

<img width="1111" alt="Screen Shot 2022-08-16 at 5 52 29 PM" src="https://user-images.githubusercontent.com/59845076/184992530-f9c5e3d1-8b63-449c-a3bc-90c687507678.png">

Reply:
Oh I probably didnt notice that when I was playing around locally - keep the div then! XD Sorry!

Reply:
Removed unnecessary styles!